erc20-bridge-token/hardhat.config.js

```diff
@@ -151,7 +151,7 @@ module.exports = {
   solidity: {
     compilers: [
       {
-        version: '0.8.11',
```

Comment:
Let's add this config:
```
            metadata: {
                // do not include the metadata hash, since this is machine dependent
                // and we want all generated code to be deterministic
                // https://docs.soliditylang.org/en/v0.8.11/metadata.html
                bytecodeHash: "none"
            }
```

Reply:
5cc943f443779cedad24be5f7360bfc3d2550db9